x-pack/plugins/ml/server/models/file_data_visualizer/import_data.js

```diff
@@ -45,7 +44,7 @@ export function importDataProvider(callWithRequest) {
       if (data.length && indexExits(index)) {
         const resp = await indexData(index, createdPipelineId, data);
         if (resp.success === false) {
-          if (resp.failures.length === data.length) {
+          if (resp.ingestError/* || resp.failures.length === data.length*/) {
```

Comment:
Just checking you meant to leave this comment out code in? Is it for a future but of work?